Tests/test_file_fli.py

```diff
@@ -32,6 +35,24 @@ def test_sanity() -> None:
         assert im.is_animated
 
 
+def test_flc_with_prefixchunk_sanity() -> None:
```

Comment:
```suggestion
def test_flc_with_prefix_chunk() -> None:
```
`test_sanity` is the name we use for a test method at the start that checks basic functionality. You don't need to use 'sanity' in any other method.